### ISSUE TITLE
Issue with erasing ROMS partition

### ISSUE THREAD

**Issue by bluest2** [skyzyx/rg35xx-garlicos-macos-instructions#12]
Hi, firstly, thank you for providing this guide. I have an issue when I erase the ROMS partition in Disk Utility the operation fails with the following message: 

_**Erasing "ROMS" (disk6s4) and creating "ROMS"**

Unmounting disk
Mediakit reports not enough space on device for requested operation.: (-5344)

**Operation failed...**_

I notice that my ROMS partition size is much larger than in the example in your tutorial and in fact the combined size of ROMS and MISC partitions is a little larger than the total SD card capacity (64GB card).

**Comment by dwiwad** [comment]
Same here! No idea how to fix it.

**Comment by justinschoi93** [comment]
I'm experiencing the same issue. Does anybody even look at these?

**Comment by skyzyx** [comment]
> Does anybody even look at these?

First of all, don't be entitled. I put these guides together to help, but that's not the same as providing support for any issue people run into. Be cool.

@bluest2 asked:

> I have an issue when I erase the ROMS partition in Disk Utility the operation fails with the following message:

I'm going to need WAY more information so that I can replicate your issue.

1. Which tutorial are you reading?
2. Which step are you on?
3. What details can you provide about your SD card?
4. You said:

    > I notice that my ROMS partition size is much larger than in the example in your tutorial and in fact the combined size of ROMS and MISC partitions is a little larger than the total SD card capacity (64GB card).

    More details here, please.

**Comment by justinschoi93** [comment]
I've followed your instructions, step by step, exactly as posted. I've...

1. Downloaded and unzipped the file
2. Flashed the img onto my SD card (samsung 128gb)
3. Renamed partitions
4. Backed up contents of ROMS partition

But when I try to erase the ROMS partition, I keep getting the following error. 
![ErasingRomsError](https://github.com/skyzyx/rg35xx-garlicos-macos-instructions/assets/122292721/17538466-6179-401c-82a0-15e2e1582716)

May I ask for some assistance?

Warm regards,
Justin

**Comment by skyzyx** [comment]
Searching for the message “MediaKit reports not enough space on device for requested operation” turned up some links:

* https://apple.stackexchange.com/questions/317048/mediakit-reports-not-enough-space-on-device-for-requested-operation
* https://appuals.com/fix-mediakit-reports-not-enough-space-on-device-for-requested-operation/
* https://superuser.com/questions/233531/how-can-i-resolve-the-error-mediakit-reports-partition-map-too-small

I'm going to erase my card and go through the instructions again to see if I can replicate this issue. I've never run into it before.

**Comment by skyzyx** [comment]
Added a new section called [Prepare your SD card](https://github.com/skyzyx/rg35xx-garlicos-macos-instructions/blob/main/docs/installing-garlicos-single-card.en_us.md#prepare-your-sd-card) to the installation instructions.

**If that does not work**, then try [MediaKit reports not enough space on device for requested operation](https://github.com/skyzyx/rg35xx-garlicos-macos-instructions/blob/main/docs/media-kit-reports-not-enough-space-on-device-for-requested-operation.md). But try the first one before trying the second one.

Note that I still cannot replicate this issue. So this is a best-effort by me based on some web searches I've done. I wrote this tutorial by following these instructions myself, so I expect these instructions to work for others.
